packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_view.dart

```diff
@@ -18,35 +22,46 @@ class PropertyEditorSidebar extends StatelessWidget {
       children: [
         Text('Property Editor', style: Theme.of(context).textTheme.titleMedium),
         const PaddedDivider.noPadding(),
-        const _PropertiesList(),
+        _PropertiesList(controller: controller),
       ],
     );
   }
 }
 
 class _PropertiesList extends StatelessWidget {
-  const _PropertiesList();
+  const _PropertiesList({required this.controller});
 
-  static const itemPadding = densePadding;
+  final PropertyEditorController controller;
 
+  static const itemPadding = densePadding;
```

Comment:
add back new line after this field to separate class members

Reply:
oops, thanks!